src/templates/utils/stylingUtils.ts

```diff
@@ -0,0 +1,32 @@
+import classNames from "classnames"
+import _ from "lodash"
+
+const OVERRIDDEN_CLASS_NAMES = [
+  "bp-section",
+  "bp-container",
+  "bp-sec-button",
+  "is-inverted",
+  "row",
+  "col",
+  "sgds-icon",
+]
+const UNSTYLED_CLASS_NAMES = [
+  "has-text-secondary",
+  "bg-secondary",
+  "is-secondary",
+]
+
+interface StylesProps {
+  [key: string]: string
+}
+
+export const getClassNames = (styles: StylesProps, classes: string[]) => {
```

Comment:
nit: you could use `Record<string, string>` here

Reply:
Fixed in https://github.com/isomerpages/isomercms-frontend/pull/895/commits/bd1b4ec8b9b33e76d0d3de23bf049da198b87bfc